src/b.cmd

```diff
@@ -0,0 +1 @@
+msbuild /v:m /p:TargetGroup=uap %1\src\%1.csproj /flp:v=diag %2 %3 %4
```

Comment:
haha no this was something that I was using locally to build faster, didn't meant to check it in. I'll remove it.